packages/fuel-gauge/src/payable-annotation.test.ts

```diff
@@ -15,11 +16,17 @@ const setupContract = createSetupConfig({
   abi: abiJSON,
 });
 
-test('allow sending coins to payable functions', async () => {
-  const contract = await setupContract();
+let contract: Contract;
+let gasPrice: BN;
+
+beforeAll(async () => {
+  contract = await setupContract();
+  ({ minGasPrice: gasPrice } = contract.provider.getGasConfig());
+});
 
+test('allow sending coins to payable functions', async () => {
   // This should not fail because the function is payable
-  expect(
+  await expect(
```

Comment:
Oof. These are fun to catch. 😰 

How could we get an `eslint` warning for missing `await`s?

Another test in the same file asserts an `async` method; shouldn't this be like that?

```suggestion
  await expect(async () =>
```

Reply:
@arboleya It seems that the proper way to handle this assertion is without `async () =>`.

Jest complains that a promise was expected but it received a function instead.

![Screenshot from 2023-10-13 10-09-26](https://github.com/FuelLabs/fuels-ts/assets/30977845/7707927b-5f47-42ec-af22-2ccff0745a7f)